x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/tabs/summary_tab/helpers.ts

```diff
@@ -36,7 +36,7 @@ export const getSummaryData = (
   { categoryId: 'ecs-compliant', mappings: partitionedFieldMetadata.ecsCompliant.length },
 ];
 
-export const getFillColor = (categoryId: CategoryId): string => {
+export const getFillColor = (categoryId: CategoryId | string): string => {
```

Comment:
Since you have a default, catch-all, value instead of adding a type assertion on the argument or check the argument before passing it, I've extended the type of the argumento to accept a string too. This is anyway is captured by the default case